python/kvikio/_lib/libkvikio.pyx

```diff
@@ -30,10 +28,10 @@ cdef class IOFuture:
         return is_future_done(self._handle)
 
 
-cdef IOFuture _wrap_io_future(future[size_t] &future):
+cdef IOFuture _wrap_io_future(future[size_t] &fut):
```

Comment:
`cython-lint` noticed that the argument `future` was shadowing the type `future`. This rename fixes it.